client/src/AboutUs.tsx

```diff
@@ -1,7 +1,272 @@
+        width: '100%',
+        height: '100%',
+    },
+    whyImagePlaceholder: {
+        width: '100%',
+        maxWidth: '474px',
+        height: '300px',
+        border: '1px solid black',
+        backgroundColor: '#C4C4C4',
+    },
+    biosImagePlaceholder: {
+        width: '168px',
+        height: '168px',
+        border: '1px solid black',
+        backgroundColor: '#C4C4C4',
+    },
+    missionContainer: {
+    },
+    biosContainer: {
+        '& $headerText': {
+            marginBottom: '20px',
+        },
+        '& $titleText': {
+        },
+    },
+    biosImagesContainer: {
+        width: '100%',
+        maxWidth: '1200px',
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'center',
+        alignItems: 'center',
+        flexWrap: 'wrap',
+    },
+    biosItem: {
+        padding: '15px',
+    },
+    trustContainer: {
+        backgroundColor: '#EBEBEB',
+    },
+    trustContent: {
+        position: 'relative',
+        zIndex: 10,
+    },
+    trustImage: {
+        width: '90%',
+        maxWidth: '1100px',
+        height: '389px',
+        border: '1px solid black',
+        backgroundColor: '#C4C4C4',
+        margin: '40px 0 20px 0',
+    },
+    buttonRow: {
+        width: '100%',
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'space-around',
+        flexWrap: 'wrap',
+        paddingTop: '30px',
+        paddingBottom: '10px',
+        maxWidth: '1200px',
+        '& > div': {
+            margin: '10px',
+        }
+    },
+    guidelinesBGWrapper: {
+        position: 'relative',
+        width: '100%',
+        height: '0',
+        zIndex: 2,
+        '& svg': {
+            position: 'absolute',
+            top: '-300px',
+            bottom: '0',
+            left: '0',
+            width: '100%',
+            height: '300px',
+            fill: '#F8F8F8',
+        },
+    },
+    orgContainer: {
+    },
+    orgContent: {
+        width: '100%',
+        maxWidth: '1200px',
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'center',
+        alignItems: 'center',
+        flexWrap: 'wrap',
+        marginTop: '20px',
+    },
+    orgImage: {
+        width: '60px',
+        height: '60px',
+        border: '1px solid black',
+        backgroundColor: '#000000',
+        borderRadius: '50%',
+        margin: '30px 80px 30px 80px',
+    },
+    titleText: {
+        fontSize: '1.3rem',
+        marginTop: '6px',
+    },
+}));
 
 function AboutUs() {
-    return <div>AboutUs</div>;
+    
+    const classes = useStyles();
+    const bios = ['Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1', 'Team1'];
+    const orgs = ['Google', 'Facebook', 'Chrome', 'Org', 'Org', 'Org', 'Org', 'Org', 'Org', 'Org'];
+
+    return (
+        <>
+            <Box className={`${classes.titleBox} ${classes.mainPageSection}`}>
+            </Box>
```

Comment:
i know it's in the mocks but let's keep this awkward giant whitespace out of this for now
(it just looks like there's nothing on the page on page load)
maybe keep this box and style commented out right above the `return` so we keep the styling for whenever they fill that space